### ISSUE TITLE
[ENHANCEMENT] Make Rebels shoot in rapid fire. 

### ISSUE THREAD

**Issue by StanleySweet** [JACoders/OpenJK#770]
In Jedi Knight : Jedi Outcast you allies would shoot in bursts instead of bullet per bullet. That made them a bit more efficient, it would be nice to have that behaviour back, been it a different option in the NPCs.CFG file

**Comment by ensiform** [comment]
The behaviors could be changed but if it requires making changes to files shipped with the game then this would be out of the scope of the project.  No asset changes are part of this project.

**Comment by StanleySweet** [comment]
Yeah I understood that, I believe it's in the C++ code related to AI or
something because enemies seems to use burstfire.

2015-12-22 19:14 GMT+01:00 Ensiform <notifications@github.com>:

**Comment by ensiform** [comment]
The other issue I see is that it becomes a gameplay change too.
